src/features/user/Settings/EditProfile/EditProfileForm.tsx

```diff
@@ -287,32 +307,40 @@ export default function EditProfile({}: Props) {
             </div>
           </label>
         </div>
+
         {type !== 'tpo' ? (
-          <MaterialTextField
-            label={t('editProfile:iamA')}
-            variant="outlined"
-            select
-            defaultValue={type}
-          >
-            {profileTypes.map((option) => (
-              <MenuItem
-                key={option.value}
-                value={option.value}
-                onClick={() => setAccountType(option.value)}
-              >
+          <MuiAutoComplete
+            id="profile-type"
+            value={localType}
+            options={profileTypes}
+            getOptionLabel={(option) => option.title}
+            isOptionEqualToValue={(option, selectedOption) =>
+              option.value === selectedOption.value
+            }
+            renderOption={(props, option) => (
+              <StyledAutoCompleteOption {...props} key={option.id}>
                 {option.title}
-              </MenuItem>
-            ))}
-          </MaterialTextField>
+              </StyledAutoCompleteOption>
+            )}
+            onChange={(event, newType) => {
+              if (newType) {
+                setAccountType(newType.value);
+                setLocalType(newType);
+              }
+            }}
+            renderInput={(params) => (
+              <TextField {...params} label={t('editProfile:iamA')} />
+            )}
+          />
         ) : null}
```

Comment:
You could also use the `<ReactHookFormSelect>` component that is used in BankDetailsForm and PayoutScheduleForm.

Reply:
Left as is as this needs a greater refactor to the code of the `ReactHookFormSelect` component to make it more generic.